### PR TITLE
fix issue-296 for discrete and continuous columns

### PR DESCRIPTION
**Pull request by R-Palazzo**

### CODE REVIEW

**Comment by R-Palazzo** [comment]
Resolves #296

File: sdmetrics/reports/utils.py [hunk 1]
```diff
@@ -86,7 +86,8 @@ def make_discrete_column_plot(real_column, synthetic_column, sdtype):
     synthetic_data['Data'] = 'Synthetic'
 
     missing_data_real = round((real_column.isna().sum() / len(real_column)) * 100, 2)
-    missing_data_synthetic = round((synthetic_column.isna().sum() / len(synthetic_column)), 2)
+    missing_data_synthetic = round(
```
Comment (amontanez24) [inline sdmetrics/reports/utils.py:89]: I think one way we can avoid bugs like this in the future is by pulling all the logic into on common placce. I think it makes sense to have a private static method `_get_missing_percentage` that takes in the column and returns the percentage and then we can just call it on lines 88, 89, 157 and 158

File: sdmetrics/reports/utils.py [hunk 2]
```diff
@@ -154,7 +155,8 @@ def make_continuous_column_plot(real_column, synthetic_column, sdtype):
     """
     column_name = real_column.name if hasattr(real_column, 'name') else ''
     missing_data_real = round((real_column.isna().sum() / len(real_column)) * 100, 2)
-    missing_data_synthetic = round((synthetic_column.isna().sum() / len(synthetic_column)), 2)
+    missing_data_synthetic = round(
```
Comment (amontanez24) [inline sdmetrics/reports/utils.py:158]: Can we also add a check in the unit tests to make sure this is correct now?

**Review by amontanez24 — CHANGES_REQUESTED** [review]

File: sdmetrics/reports/utils.py (hunk 2 above)
Comment (R-Palazzo) [reply sdmetrics/reports/utils.py:158]: Yes I can

File: sdmetrics/reports/utils.py (hunk 1 above)
Comment (R-Palazzo) [reply sdmetrics/reports/utils.py:89]: Yes, it's a good idea, I can do this

File: sdmetrics/reports/utils.py [hunk 3]
```diff
@@ -153,8 +154,8 @@ def make_continuous_column_plot(real_column, synthetic_column, sdtype):
         plotly.graph_objects._figure.Figure
     """
     column_name = real_column.name if hasattr(real_column, 'name') else ''
-    missing_data_real = round((real_column.isna().sum() / len(real_column)) * 100, 2)
-    missing_data_synthetic = round((synthetic_column.isna().sum() / len(synthetic_column)), 2)
+    missing_data_real = round(get_missing_percentage(real_column), 2)
```
Comment (amontanez24) [inline sdmetrics/reports/utils.py:157]: I think even the rounding part could go in the `get_missing_percentage` function

**Review by amontanez24 — APPROVED** [review]
One small comment but after that :shipit:

File: sdmetrics/reports/utils.py (hunk 3 above)
Comment (R-Palazzo) [reply sdmetrics/reports/utils.py]: Moved the rounding part inside the function.

**Review by amontanez24 — APPROVED** [review]
:shipit:

**Pull request merged**